### PR TITLE
Danish Language Support

### PR DESCRIPTION
**Pull request by teknoabu**
Translated the whole page into Danish and added it as a language option on the homepage.

### CODE REVIEW

**Comment by teknoabu** [comment]
Currently replacing somehow misplaced "" with ''.

**Comment by peppy** [comment]
Please stop making individual commits

**Comment by teknoabu** [comment]
Finally, everything should be in place!

**Comment by kj415j45** [comment]
better ask for some reviews

File: resources/lang/da/home.php [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+        ],
+
+        'wiki_page' => [
+            'link' => 'Søg på wikiet',
+            'more_simple' => 'Se flere wiki resultater',
+            'title' => 'Wiki',
+        ],
+    ],
+
+    'download' => [
```
Comment (Padnezz) [inline resources/lang/da/home.php]: This is the old array for the download page, check my pull request #2080 or `resources/lang/en/home.php` for the new array

Comment (teknoabu) [reply resources/lang/da/home.php]: Thanks for letting me know!

File: resources/lang/da/supporter_tag.php [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+ *    along with osu!web.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+return [
+    'user_search' => [
+        'searching' => 'søger...',
+        'not_found' => 'Denne bruger eksisterer ikke',
+    ],
+    'duration' => [
+        'months' => ':length måned|:length måneder|{0}',
```
Comment (notbakaneko) [inline resources/lang/da/supporter_tag.php]: we moved these to `common.count.*`

**Pull request merged**